src/engraving/dom/stem.h

```diff
@@ -67,9 +68,11 @@ class Stem final : public EngravingItem
     Millimetre userLength() const { return m_userLength; }
     void setUserLength(Millimetre userLength) { m_userLength = userLength; }
 
-    Millimetre lineWidth() const { return m_lineWidth; }
-    double lineWidthMag() const { return m_lineWidth * mag(); }
-    void setLineWidth(Millimetre lineWidth) { m_lineWidth = lineWidth; }
+    virtual double point(Spatium sp) const { return sp.val() * score()->style().spatium(); }
```

Comment:
Shouldn't that be
```suggestion
    double point(Spatium sp) const override { return sp.val() * score()->style().spatium(); }
```
Also, maybe it is better to keep it in .cpp (also because it requires including score.h). But I have no strong opinion on that.